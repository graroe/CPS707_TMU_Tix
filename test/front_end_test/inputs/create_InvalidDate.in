login
admin
create
NewEvent
2023DC01
202312001
12012023
20231003
20251004
20231131
!q
logout