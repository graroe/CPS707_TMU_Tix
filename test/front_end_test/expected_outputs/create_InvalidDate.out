Enter session type <sales or admin>: 
Admin login successful.
Enter new event name <max 15 chars>: 
Enter date of event <format YYYYMMDD>: 
Not a valid date.\nEnter date of event <format YYYYMMD>: 
Not a valid date.\nEnter date of event <format YYYYMMD>: 
Not a valid date.\nEnter date of event <format YYYYMMD>: 
Not a valid date.\nEnter date of event <format YYYYMMD>: 
Not a valid date.\nEnter date of event <format YYYYMMD>: 
Not a valid date.\nEnter date of event <format YYYYMMD>: 
Transaction canceled.
Session terminated.